src/ErriezDHT22.cpp

```diff
@@ -102,9 +102,9 @@ void DHT22::begin(uint8_t maxReadRetries, uint8_t numSamples)
  *      The application should call this function and check if a new temperature and humidity can be
  *      read to prevent too fast sensor reads.
  * \retval true
- *      Available, interval between sensor reads >= 2000 ms.
+ *      Available, interval between sensor reads >= 2000 ms and sensor read was successful.
  * \retval false
- *      Not available, interval between sensor reads too short.
+ *      Not available, interval between sensor reads too short, or read failed.
  */
 bool DHT22::available()
```

Comment:
This name is no longer sufficient for the implementation. AvailableAndRead should be a better name I guess.

The same applies to the \brief of it's doxygen header.